CRM/MembershipExtras/Helper/PaymentScheduleAPI.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+
+class CRM_MembershipExtras_Helper_PaymentScheduleAPI {
+
+  /**
+   * Validates the schedule param
+   *
+   * @param array $params
+   * @throws API_Exception
+   */
+  public function validateSchedule(array $params) {
+    if (!in_array($params['schedule'], [
+      CRM_MembershipExtras_Service_MembershipInstalmentsSchedule::MONTHLY,
+      CRM_MembershipExtras_Service_MembershipInstalmentsSchedule::QUARTERLY,
+      CRM_MembershipExtras_Service_MembershipInstalmentsSchedule::ANNUAL,
+    ])) {
+      throw new API_Exception('Schedule is not of monthly, quarterly or annual');
+    }
+  }
+
+  /**
+   * Gets instalment from the API params and membership types
+   *
+   * @param array $params
+   * @param array $membershipTypes
+   * @param array $nonMembershipPriceFieldValue
+   * @return array
+   * @throws CRM_MembershipExtras_Exception_InvalidMembershipTypeInstalmentCalculator
+   * @throws Exception
+   */
+  public function getInstalments(array $params, array $membershipTypes, array $nonMembershipPriceFieldValue = []): array {
```

Comment:
can you please get rid of all return type declaration in this PR, my reasons are : 

- you are doing it in some places but leaving others in this PR
- we did not communicate together as a team if we want to start doing that.
- we also use @return in docblocks, I know it is not exactly the same but it still duplicates information.

 I just don't see it as a right step to be done at this point, and there are other important improvements we need to work on before looking at things like this and other stuff like static typing..etc